tox.ini

```diff
@@ -14,12 +15,13 @@ setenv =
     DRFG_TEST_RUNNER=./tests/manage.py test
     pytest: DRFG_TEST_RUNNER=-m pytest
 commands =
-    coverage run {env:DRFG_TEST_RUNNER} {posargs:tests/django_restframework_gis_tests} --parallel
+    coverage run {env:DRFG_TEST_RUNNER} {posargs:tests/django_restframework_gis_tests}
```

Comment:
Why this parallel flag is removed? Kindly follow the guidelines
https://openwisp.io/docs/dev/developer/contributing.html#avoiding-unnecessary-changes